app/api/webhooks/paypal.py

```diff
@@ -161,26 +161,28 @@ async def process_notification(
             pass
 
     notification = dict(request_params)
+    transaction_id = notification["txn_id"]
```

Comment:
Confirmed this should always be available here via production data:

https://one.newrelic.com/logger?account=4105649&duration=604800000&state=1d63150b-316f-4a68-8f4f-0ef3efb590e5